roles/keystone/tasks/main.yml

```diff
@@ -31,6 +31,15 @@
   notify:
     - restart keystone services
 
+- name: install ldap pip reqs in package venv
+  command: "{{ 'keystone'|ursula_package_path(openstack_package_version) }}/bin/pip install {{ item }}"
+  register: pipout
+  changed_when: pipout.stdout|search("Successfully installed")
+  notify: restart keystone services
+  when: openstack_install_method == 'package' and
+        openldap.enabled|bool
+  with_items: keystone.package.ldap_python
```

Comment:
turned out that installing using sourced needed some extra dev packages for pip so I had to add an extra handler in the openstack-source role.